### PR TITLE
Feat/filtering ingredients and measures for current drink

### PR DESCRIPTION
**Pull request by raz-joan**
What (if any) features are you implementing?
- Added structure to PairedPage Component
- Created Drink and Game Components
- Add hooks for accessing global state in the above components
- Add dispatch for favorite drink onClick of the button on the PairingForm
- Update fetch to store currentDrink object instead of both randomDrink and currentDrink
- Add methods to extract ingredient and measure information from currentDrink
- 
What (if anything) did you refactor?
- Removed unnecessary and commented out code
- Fixed quotations to single instead of double
- Updated fetch for currentDrink and corresponding dispatch
- No longer storing randomDrink in state --> still need to remove action / reducer
- 
Were there any issues that arose?
- issues with timing with async functions and storing info to state
- 
Is there anything that you need from your teammate?
- n/a
Any other comments, questions, or concerns?
- still killin' it.

### CODE REVIEW

File: src/Components/Drink.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    const ingredientsList = recipe.ingredients.map((ingredient, index) => {
+        return (
+            <p>{`${recipe.measures[index]} ${ingredient}`}</p>
+        )
+    })
+
+    return (
+        <div className='paired-component'>
+            <h3>{currentDrink.strDrink}</h3>
+            <img class='drink-image' src={currentDrink.strDrinkThumb} />
```
Comment (michelecomfort) [inline src/Components/Drink.js:61]: class --> className
We can add our alt-tags to images

File: src/Components/Drink.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            for (let i = 0; i < difference; i++) {
+                measures.push('')
+            }
+            return {ingredients, measures}
+        } else {
+            return {ingredients: [], measures: []}
+        }
+    }
+
+    const createIngredientAmount = (currentDrink) => {
```
Comment (colganmeanor) [inline src/Components/Drink.js:43]: Joan - such a good job on creating these functions. The logic seems really solid to me. I'm not sure of ways to clean this up or shorten this code at this exact moment but I'm sure we'll all be thinking of this as we finish up our work.

**Review by michelecomfort — COMMENTED** [review]
Great work team!

**Review by colganmeanor — COMMENTED** [review]
everything looking good yall - thanks all for your hard work

**Pull request merged**